### ISSUE TITLE
Instagram posts: cron jobs possibly not working, caching issue

### ISSUE THREAD

**Issue by JonEsparaz** [themeetinghouse/mobile#100]
Two main issues:

* From what I can tell, the cron jobs are not working since new posts aren't being pulled. Perhaps we can setup the cron jobs on another machine to see if the issue if specific to the MacOS server?
* Instagram appears to clear their caches every so often, so the image URIs for older posts expire. To fix: edit [`getPostsForLocation()`](https://github.com/themeetinghouse/web/blob/0e279c42be305069be6071e3129d9667f99ea2e8/src/pages/admin/instagram.tsx#L53) so that the `thumbnails` field gets updated on every pull (but this is likely costly in terms of DynamoDB writes).

┆Issue is synchronized with this [Wrike Task](https://www.wrike.com/open.htm?id=588046515)

**Comment by JonEsparaz** [comment]
@lucastbelem someone should fix this before the launch. If it can't get done, I think the Instagram module should just be hidden for now.

**Comment by GeorgeBellTMH** [comment]
@branden-eyk can you check on the cron job?

**Comment by branden-eyk** [comment]
Sorry for the delay! I had another look at the cron jobs and I think I figured out why they aren't working properly and why it originally seemed to me like they were: the cron jobs DO run, but they don't seem to recognize node as a command (despite node working fine if you run it). So I did some Googling and the solution is to create a bash script that has the node command you want to run and set the jobs to run that script rather than the command directly. I've modified the jobs accordingly. We'll see how it goes when it runs later tonight and tomorrow morning.
